backend/src/services/org/org-service.ts

```diff
@@ -118,6 +131,54 @@ export const orgServiceFactory = ({
     return workspaces.filter((workspace) => organizationWorkspaceIds.has(workspace.id));
   };
 
+  const addGhostUser = async (orgId: string, tx?: Knex) => {
+    const email = `ghost-${alphaNumericNanoId(16)}-${orgId}@infisical.com`; // We add a nanoid because the email is unique. And we have to create a new ghost user each time, so we can have access to the private key.
```

Comment:
what do you think about calling it `sudo-${alphaNumericNanoId(16)}-${orgId}@infisical.com`. It might sound a bit less scary and when we discuss this in the context of debugging with customers. Instead of saying `ghost` user we can refer to it as `sudo` user. This is your call though, just a suggestion

Reply:
I agree!